src/shared/components/Input/Input.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import './style.scss';
 import PropTypes from 'prop-types';
 
-const InputComponent = ({ register, className, placeholder, type, name, onChange }) => {
+const InputComponent = ({ value, register, className, placeholder, type, name, onChange }) => {
```

Comment:
Do you pass value to `InputComponent` anywhere?

Reply:
Done